tests/cases/conformance/externalModules/typeOnly/noErasingImportedNames.ts

```diff
@@ -0,0 +1,28 @@
+// @preserveValueImports: true
```

Comment:
The name for this test (as well as the `_errors.ts` and `_module.ts` variations) should be updated to match the compiler flag name to more easily find the tests for the feature in the future.